src/ontology/cl-edit.owl

```diff
@@ -5463,14 +5463,14 @@ SubClassOf(obo:CL_0000209 obo:CL_0002076)
 
 # Class: obo:CL_0000210 (photoreceptor cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D010786") obo:IAO_0000115 obo:CL_0000210 "A cell specialized to detect and transduce light.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D010786") obo:IAO_0000115 obo:CL_0000210 "A cell specialized in detecting light stimuli that are involved in visual perception.")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000210 "BTO:0001060")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000210 "CALOHA:TS-0868")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000210 "FMA:86740")
 AnnotationAssertion(rdfs:label obo:CL_0000210 "photoreceptor cell")
 AnnotationAssertion(foaf:depiction obo:CL_0000210 "https://www.swissbiopics.org/api/image/Photoreceptor_cell.svg"^^xsd:anyURI)
-EquivalentClasses(obo:CL_0000210 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0050962)))
-SubClassOf(Annotation(oboInOwl:is_inferred "true") obo:CL_0000210 obo:CL_0000006)
+EquivalentClasses(obo:CL_0000210 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0050908)))
+SubClassOf(obo:CL_0000210 obo:CL_0008028)
```

Comment:
I think this change caused the inferred equivalence, which is not allowed and it's the reason the QC is failing. :) 

```bash
2024-08-01 12:28:49,500 ERROR org.obolibrary.robot.ReasonOperation - Only equivalent classes that have been asserted are allowed. Inferred equivalencies are forbidden.
2024-08-01 12:28:49,502 ERROR org.obolibrary.robot.ReasonOperation - Equivalence: <http://purl.obolibrary.org/obo/CL_0000488> == <http://purl.obolibrary.org/obo/CL_0000210>
```

Reply:
I will approve the PR after Anita's changes resolved :)

Reply:
David said he fixed that in GO. We're waiting for a GO release and will update the imports then.